code_strategy.md

```diff
@@ -0,0 +1,110 @@
+We are biomedical data scientists who use code to answer biological hypotheses and explore biological data. It is therefore important to define how we use, write, and share code. 
+
+## 1. Using code
+
+Code is ubiquitous on the internet, and there are many examples of both elegant and useless code. We leverage all possible resources to achieve our goals; using both trusted software and validated code snippets where available.
+
+We use trusted software wherever possible, selecting to use code that is tried and true, with a strong, active community. When we use these software packages, we specifically take note of which version we’re using, as different versions of different software have different functionality, and often induce different results.
+
+We indicate and track these versions in environment files (for example, [here](https://github.com/broadinstitute/cell-health/blob/master/environment.yml#L6)). When we write papers, we are also conscientious to cite all pieces of software we used in our analyses.
+
+We also often use code snippets that we identify in Stack Overflow, github gists, package tutorials, other codebases in the lab, or other sources. These pieces of code present examples on how to perform specific tasks, and we typically use them through some combination of copy+paste+modify. When “borrowing code”, we make sure to specifically cite where we derived this inspiration from, using in-line comments (for example, [here](https://github.com/cytomining/pycytominer/blob/master/pycytominer/operations/transform.py#L16)). We are cautious in using these examples, validating that they work specifically for our use-case.
+
+Most of the time, it is safe to assume that all code you can view publicly can be used freely. However, we are also aware that certain licenses might exist that prevent us from repurposing other people’s code. We must look out for these exceptions, and steer clear from using code with restrictive licenses.
+
+## 2. Writing code
+
+We take pride in the code we write. All code we produce should 1) work over and over again, 2) be easily understood, 3) be well organized.
+
+### 1. Code should work over and over again (repeatable)
+
+Good code is repeatable. Running the piece of code should return expected results today, tomorrow, next month, and next year for anyone who runs the code (especially you!). This means specifying software versions in an environment file (or docker image), setting random seeds (where randomization needs to be avoided), documenting input and output data-specific formats, and version control (for both data and code).
+
+### 2. Code should be easily understood (readable)
+
+Good code is readable. Based on the logic, variable names, and documentation, anyone (especially you!) should be able to understand what your code is doing. Readability also means standard formatting. We always lint our code, and use automatic formatting wherever possible (e.g. [python black](https://github.com/psf/black)). 
```

Comment:
Recommend mentioning `pre-commit` as well where we can specify linters such as black to also ensure consistent format and reproducibility.

Reply:
Done in https://github.com/WayScience/playbooks/pull/1/commits/500e19465c660484531077595340603a15f2570b